package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/editorjs",
-  "version": "2.18.0",
+  "version": "2.18.1",
```

Comment:
2.18 is not released yet, so you can include patch to 2.18 instead of bumping patch version

Reply:
Oh ok